manifests/capacityscheduling/scheduler-config.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: kubescheduler.config.k8s.io/v1alpha2
+kind: KubeSchedulerConfiguration
+leaderElection:
+  leaderElect: false
+clientConnection:
+  kubeconfig: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
+profiles:
+- schedulerName: default-scheduler
+  plugins:
+    preFilter:
+      enabled:
+        - name: CapacityScheduling
```

```suggestion
      - name: CapacityScheduling
```

(applies elsewhere in yamls)

---

How about replacing all `CapacityScheduling` with `ElasticQuota`?

---

Thanks  I replace `CapacityScheduling ` with `ElasticQuota` in api.  And the name of plugin keeps as `CapacityScheduling`. It maybe like `podgroup` in `coscheduling`.
@yuanchen8911